<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Personal budget Tracker</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="filterLabel">
       <property name="text">
        <string>Filter: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="filterLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="columnLabel">
       <property name="text">
        <string>Column: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="filterColumnComboBox"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="addExpenseButton">
       <property name="text">
        <string>Add Expense</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeExpenseButton">
       <property name="text">
        <string>Remove Expense</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
